qemu/tests/block_resize.py

```diff
@@ -73,7 +73,12 @@ def compare_block_size(session, block_cmd, block_pattern):
 
     drive_path = ""
     if params.get("os_type") == 'linux':
-        drive_id = params["blk_extra_params_%s" % data_image].split("=")[1]
+        pattern = r"(serial|wwn)=(\w+)"
+        match = re.search(pattern, params["blk_extra_params_%s" % data_image], re.M)
+        if match:
+            drive_id = match.group(2)
+        else:
+            raise test.fail("No available tag to get drive id")
```

Comment:
no raise, use test.fail directly

Reply:
thanks, will update it.

Reply:
for line 9 image_snapshot_image1 = yes, as I know, snapshot don't support resize